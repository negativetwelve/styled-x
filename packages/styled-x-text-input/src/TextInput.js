// Libraries
import Styled from 'styled-x-primitives';
import ReactXTextInput from 'react-x-text-input';

const TextInput = Styled(ReactXTextInput)`
`;

export default TextInput;
