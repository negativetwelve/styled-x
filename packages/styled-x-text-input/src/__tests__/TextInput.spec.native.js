// Modules
import React from 'react';
import TextInput from '../TextInput';

const Input = TextInput.extend`
  font-size: 14px;
  border-radius: 3px;
  border: solid rgba(229, 229, 229, 1.0) 1px;
  padding: 12px;
`;

/* eslint-disable no-undef */
describe('TextInput`', () => {
  describe('Input', () => {
    itRenders(() => <Input />);
  });
});
