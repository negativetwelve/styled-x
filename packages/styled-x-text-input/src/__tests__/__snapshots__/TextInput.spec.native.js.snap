// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput\` Input renders snapshot 1`] = `null`;
